Assets/Mirror/Editor/Weaver/Extensions.cs

```diff
@@ -66,9 +66,11 @@ public static TypeReference GetEnumUnderlyingType(this TypeDefinition td)
             throw new ArgumentException($"Invalid enum {td.FullName}");
         }
 
-        public static bool ImplementsInterface(this TypeDefinition td, TypeReference baseInterface)
+        public static bool ImplementsInterface<Interface>(this TypeDefinition td)
```

Comment:
```suggestion
        public static bool ImplementsInterface<TInterface>(this TypeDefinition td)
```

can we use `TInterface` so it is obvious it is generic